### PR TITLE
added view for total number of alumni in Alumni Status page

### PR DESCRIPTION
**Pull request by aniarya82**

### CODE REVIEW

File: app/views/viewall/alumni.html.erb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <h2 align="center">Complete Alumni Database</h2>
+<h4 align="center">Total Count: &nbsp; <label class="label label-info"> <%= @allAlums.count %> </label></h4>
```
Comment (icyflame) [inline app/views/viewall/alumni.html.erb]: The label is not necessary here. The UI of the info label is slightly screwed up, I can't seem to make out the numbers clearly.

Comment (icyflame) [inline app/views/viewall/alumni.html.erb]: @aniarya82 
Sorry, what I meant was that the UI of badges and the color of `info` is not good.

So, please change this to plain numbers. We don't need a label here.

**Comment by aniarya82** [comment]
screen shot 2015-06-14 at 11 43 20 am

**Comment by icyflame** [comment]
@aniarya82 
yes, this is fine. Please implement the same in the `viewall` pages too.

**Pull request merged**